### PR TITLE
feat(genapi): public beta launch

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- [X] Rate limits defined
- [X] 429 errors explained
- [X] New models added
- [X] Playground explained
- [X] Vision models how to
- [X] Mentions of private beta removed
- [X] Metas revisited

### CODE REVIEW

File: ai-data/generative-apis/how-to/query-language-models.mdx [hunk 1]
```diff
@@ -1,35 +1,45 @@
...
 - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/) for API authentication
 - Python 3.7+ installed on your system
 
 ## Accessing the Playground
 
-Scaleway's Playground is in development, stay tuned!
+Scaleway provides a web playground for instruct-based models hosted on Generative APIs.
 
-## Querying text models via API
+1. Navigate to Generative APIs under the AI section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-language-models.mdx:31]:
```suggestion
1. Navigate to **Generative APIs** under the **AI** section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```

File: ai-data/generative-apis/how-to/query-language-models.mdx [hunk 2]
```diff
@@ -1,35 +1,45 @@
...
+2. Click the name of the chat model you want to try. Alternatively, click <Icon name="more" /> next to the chat model, and click **Try model** in the menu. 
+
+The web playground displays.
+
+## Using the Playground
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-language-models.mdx:36]:
```suggestion
## Using the playground
```

File: ai-data/generative-apis/how-to/query-vision-models.mdx [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+---
+
+Scaleway's Generative APIs service allows users to interact with powerful vision models hosted on the platform.
+
+<Message type="note">
+  Vision models can understand and analyze images, not generate them. 
+</Message>
+
+There are several ways to interact with vision models:
+- The Scaleway [console](https://console.scaleway.com) provides complete [playground](/ai-data/generative-apis/how-to/query-vision-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-vision-models.mdx:21]:
```suggestion
- The Scaleway [console](https://console.scaleway.com) provides a complete [playground](/ai-data/generative-apis/how-to/query-vision-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
```

File: ai-data/generative-apis/how-to/query-vision-models.mdx [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/) for API authentication
+- Python 3.7+ installed on your system
+
+## Accessing the Playground
+
+Scaleway provides a web playground for vision models hosted on Generative APIs.
+
+1. Navigate to Generative APIs under the AI section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-vision-models.mdx:35]:
```suggestion
1. Navigate to **Generative APIs** under the **AI** section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```

File: ai-data/generative-apis/how-to/query-vision-models.mdx [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+2. Click the name of the vision model you want to try. Alternatively, click <Icon name="more" /> next to the vision model, and click **Try model** in the menu. 
+
+The web playground displays.
+
+## Using the Playground
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-vision-models.mdx:40]:
```suggestion
## Using the playground
```

File: ai-data/generative-apis/how-to/query-vision-models.mdx [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+- **`messages`**: A list of message objects that represent the conversation history. Each message should have a `role` (e.g., "system", "user", "assistant") and `content`. The content is an array that can contain text and/or image objects.
+- **`temperature`**: Controls the output's randomness. Lower values (e.g., 0.2) make the output more deterministic, while higher values (e.g., 0.8) make it more creative.
+- **`max_tokens`**: The maximum number of tokens (words or parts of words) in the generated output.
+- **`top_p`**: Recommended for advanced use cases only. You usually only need to use temperature. `top_p` controls the diversity of the output, using nucleus sampling, where the model considers the tokens with top probabilities until the cumulative probability reaches `top_p`.
+- **`stop`**: A string or list of strings where the model will stop generating further tokens. This is useful for controlling the end of the output.
+
+<Message type="warning">
+ If you encounter an error such as "Forbidden 403" refer to the [API documentation](/ai-data/generative-apis/api-cli/understanding-errors) for troubleshooting tips.
```
Comment (RoRoJ) [inline ai-data/generative-apis/how-to/query-vision-models.mdx:176]:
```suggestion
 If you encounter an error such as "Forbidden 403", refer to the [API documentation](/ai-data/generative-apis/api-cli/understanding-errors) for troubleshooting tips.
```

File: ai-data/generative-apis/quickstart.mdx [hunk 7]
```diff
@@ -24,15 +24,26 @@ Hosted in European data centers and priced competitively per million tokens used
...
   - A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
   - Python 3.7+ installed on your system
 
 ## Start with the Generative APIs Playground
 
-Scaleway's Playground is in development, stay tuned!
+Scaleway provides a web playground for instruct-based models hosted on Generative APIs.
+
+### Accessing the Playground
+1. Navigate to Generative APIs under the AI section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```
Comment (RoRoJ) [inline ai-data/generative-apis/quickstart.mdx:37]:
```suggestion
1. Navigate to **Generative APIs** under the **AI** section of the [Scaleway console](https://console.scaleway.com/) side menu. The list of models you can query displays.
```

File: ai-data/generative-apis/reference-content/rate-limits.mdx [hunk 8]
```diff
@@ -1,28 +1,43 @@
...
 ## Why do we set rate limits?
 
-These limits will safeguard against abuse or misuse of Scaleway Generative APIs, helping to ensure fair access to the API with consistent performance.
\ No newline at end of file
+These limits safeguard against abuse or misuse of Scaleway Generative APIs, helping to ensure fair access to the API with consistent performance.
+
+## How can I increase the rate limits?
+
+We actively monitor usage and will improve rates based on feedback.
+If you need to increase your rate limits, please contact us via the support, providing details on the model used and specific use case.
```
Comment (RoRoJ) [inline ai-data/generative-apis/reference-content/rate-limits.mdx:43]:
```suggestion
If you need to increase your rate limits, contact us via the support team, providing details on the model used and specific use case.
```

File: ai-data/generative-apis/how-to/use-structured-outputs.mdx [hunk 9]
```diff
@@ -18,14 +18,12 @@ JSON, as a widely-used format, enables seamless integration with a variety of pl
 By specifying a response format when using the [Chat Completions API](/ai-data/generative-apis/api-cli/using-chat-api/), you can ensure that responses are returned in a JSON structure.
 There are two main modes for generating JSON: **Object Mode** (schemaless) and **Schema Mode** (deterministic, structured output).
 
-You can interact with text models in several ways:
-- Via the Scaleway [console](https://console.scaleway.com), which will soon provide a complete [playground](/ai-data/generative-apis/how-to/query-text-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
-- Via the [Chat API](/ai-data/generative-apis/how-to/query-text-models/#querying-text-models-via-api)
+There are several ways to interact with language models:
+- The Scaleway [console](https://console.scaleway.com) provides complete [playground](/ai-data/generative-apis/how-to/query-language-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
```
Comment (nerda-codes) [inline ai-data/generative-apis/how-to/use-structured-outputs.mdx:22]:
```suggestion
- The Scaleway [console](https://console.scaleway.com) provides a complete [playground](/ai-data/generative-apis/how-to/query-language-models/#accessing-the-playground), aiming to test models, adapt parameters, and observe how these changes affect the output in real-time.
```

File: ai-data/generative-apis/quickstart.mdx [hunk 10]
```diff
@@ -24,15 +24,26 @@ Hosted in European data centers and priced competitively per million tokens used
...
+2. Click the name of the chat model you want to try. Alternatively, click <Icon name="more" /> next to the chat model, and click **Try model** in the menu. 
+
+The web playground displays.
+
+### Using the Playground
+1. Enter a prompt at the bottom of the page, or use one of the suggested prompts in the conversation area.
+2. Edit the hyperparameters listed on the right column, for example the default temperature for more or less randomness on the outputs. 
+3. Switch model at the top of the page, to observe the capabilities of chat models offered via Generative APIs.
```
Comment (nerda-codes) [inline ai-data/generative-apis/quickstart.mdx:45]:
```suggestion
### Using the playground
1. Enter a prompt at the bottom of the page, or use one of the suggested prompts in the conversation area.
2. Edit the hyperparameters listed on the right column, for example the default temperature for more or less randomness on the outputs. 
3. Switch models at the top of the page, to observe the capabilities of chat models offered via Generative APIs. 
```

**Review by bene2k1 — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Comment by tgenaitay** [comment]
We're good to go monday 4th anytime @bene2k1

**Pull request merged**